test_preprocessors.py tests fails with recent pygments

when building package for Alpine Linux:
```
>>> py3-cli_helpers: Validating /home/ncopa/aports/community/py3-cli_helpers/APKBUILD...
>>> py3-cli_helpers: Cleaning up srcdir
>>> py3-cli_helpers: Cleaning up pkgdir
>>> py3-cli_helpers: Cleaning up tmpdir
>>> py3-cli_helpers: Fetching https://distfiles.alpinelinux.org/distfiles/py3-cli_helpers-2.12.0.tar.gz
/var/cache/distfiles/py3-cli_helpers-2.12.0.tar.gz: OK
>>> py3-cli_helpers: Preparing build chroot...
>>> py3-cli_helpers: Using repository http://dl-cdn.alpinelinux.org/alpine/edge/main
>>> py3-cli_helpers: Using repository /home/ncopa/packages/main
>>> py3-cli_helpers: Using repository http://dl-cdn.alpinelinux.org/alpine/edge/community
>>> py3-cli_helpers: Using repository /home/ncopa/packages/community
(  1/103) Installing musl (1.2.6-r2)
(  2/103) Installing libcrypto3 (3.5.6-r0)
(  3/103) Installing libssl3 (3.5.6-r0)
(  4/103) Installing zlib (1.3.2-r0)
(  5/103) Installing libapk (3.0.6-r0)
(  6/103) Installing ca-certificates-bundle (20251003-r0)
(  7/103) Installing apk-tools (3.0.6-r0)
(  8/103) Installing libcap2 (2.78-r0)
(  9/103) Installing libcap-getcap (2.78-r0)
( 10/103) Installing fakeroot (1.37.2-r0)
( 11/103) Installing scanelf (1.3.9-r1)
( 12/103) Installing musl-utils (1.2.6-r2)
( 13/103) Installing libgcc (15.2.0-r3)
( 14/103) Installing libstdc++ (15.2.0-r3)
( 15/103) Installing lzip (1.26-r0)
( 16/103) Installing openssl (3.5.6-r0)
( 17/103) Installing patch (2.8-r0)
( 18/103) Installing pkgconf (2.5.1-r0)
( 19/103) Installing acl-libs (2.3.2-r1)
( 20/103) Installing tar (1.35-r5)
( 21/103) Installing busybox (1.37.0-r31)
  Executing busybox-1.37.0-r31.post-install
( 22/103) Installing ssl_client (1.37.0-r31)
( 23/103) Installing busybox-binsh (1.37.0-r31)
( 24/103) Installing abuild (3.17.0_rc1-r0)
  Executing abuild-3.17.0_rc1-r0.pre-install
  Installing file to etc/abuild.conf.apk-new
( 25/103) Installing abuild-sudo (3.17.0_rc1-r0)
( 26/103) Installing alpine-baselayout-data (3.7.2-r1)
( 27/103) Installing alpine-baselayout (3.7.2-r1)
  Executing alpine-baselayout-3.7.2-r1.pre-install
  Executing alpine-baselayout-3.7.2-r1.post-install
( 28/103) Installing bridge (1.5-r5)
( 29/103) Installing ifupdown-ng (0.12.1-r7)
( 30/103) Installing openrc-user (0.63-r1)
( 31/103) Installing openrc (0.63-r1)
  Executing openrc-0.63-r1.post-install
( 32/103) Installing mdev-conf (4.9-r0)
( 33/103) Installing busybox-mdev-openrc (1.37.0-r31)
( 34/103) Installing alpine-conf (3.21.0-r0)
( 35/103) Installing alpine-keys (2.6-r0)
( 36/103) Installing alpine-release (3.24.0_alpha20260127-r0)
( 37/103) Installing busybox-openrc (1.37.0-r31)
( 38/103) Installing busybox-suid (1.37.0-r31)
( 39/103) Installing alpine-base (3.24.0_alpha20260127-r0)
( 40/103) Installing jansson (2.15.0-r0)
( 41/103) Installing zstd-libs (1.5.7-r2)
( 42/103) Installing binutils (2.45.1-r1)
( 43/103) Installing libmagic (5.47-r1)
( 44/103) Installing file (5.47-r1)
( 45/103) Installing libgomp (15.2.0-r3)
( 46/103) Installing libatomic (15.2.0-r3)
( 47/103) Installing gmp (6.3.0-r4)
( 48/103) Installing isl26 (0.26-r2)
( 49/103) Installing mpfr4 (4.2.2-r0)
( 50/103) Installing mpc1 (1.3.1-r1)
( 51/103) Installing gcc (15.2.0-r3)
( 52/103) Installing libstdc++-dev (15.2.0-r3)
( 53/103) Installing musl-dev (1.2.6-r2)
( 54/103) Installing g++ (15.2.0-r3)
( 55/103) Installing make (4.4.1-r4)
( 56/103) Installing fortify-headers (3.0.1-r2)
( 57/103) Installing build-base (0.5-r4)
( 58/103) Installing hiredis (1.3.0-r0)
( 59/103) Installing libxxhash (0.8.3-r1)
( 60/103) Installing ccache (4.13.3-r0)
( 61/103) Installing libbz2 (1.0.8-r6)
( 62/103) Installing libexpat (2.7.5-r0)
( 63/103) Installing libffi (3.5.2-r0)
( 64/103) Installing gdbm (1.26-r0)
( 65/103) Installing xz-libs (5.8.2-r0)
( 66/103) Installing mpdecimal (4.0.1-r0)
( 67/103) Installing ncurses-terminfo-base (6.6_p20260404-r0)
( 68/103) Installing libncursesw (6.6_p20260404-r0)
( 69/103) Installing libpanelw (6.6_p20260404-r0)
( 70/103) Installing readline (8.3.3-r1)
( 71/103) Installing sqlite-libs (3.53.0-r0)
( 72/103) Installing python3 (3.14.3-r0)
( 73/103) Installing python3-pycache-pyc0 (3.14.3-r0)
( 74/103) Installing pyc (3.14.3-r0)
( 75/103) Installing py3-configobj-pyc (5.0.9-r2)
( 76/103) Installing python3-pyc (3.14.3-r0)
( 77/103) Installing py3-configobj (5.0.9-r2)
( 78/103) Installing py3-installer (1.0.0-r0)
( 79/103) Installing py3-installer-pyc (1.0.0-r0)
( 80/103) Installing py3-gpep517 (19-r2)
( 81/103) Installing py3-gpep517-pyc (19-r2)
( 82/103) Installing py3-iniconfig (2.3.0-r1)
( 83/103) Installing py3-iniconfig-pyc (2.3.0-r1)
( 84/103) Installing py3-parsing (3.3.2-r1)
( 85/103) Installing py3-parsing-pyc (3.3.2-r1)
( 86/103) Installing py3-packaging (26.0-r1)
( 87/103) Installing py3-packaging-pyc (26.0-r1)
( 88/103) Installing py3-pluggy (1.6.0-r1)
( 89/103) Installing py3-pluggy-pyc (1.6.0-r1)
( 90/103) Installing py3-py (1.11.0-r5)
( 91/103) Installing py3-py-pyc (1.11.0-r5)
( 92/103) Installing py3-pygments (2.20.0-r0)
( 93/103) Installing py3-pygments-pyc (2.20.0-r0)
( 94/103) Installing py3-pytest (9.0.3-r0)
( 95/103) Installing py3-pytest-pyc (9.0.3-r0)
( 96/103) Installing py3-setuptools (82.0.1-r1)
( 97/103) Installing py3-setuptools-pyc (82.0.1-r1)
( 98/103) Installing py3-wcwidth (0.6.0-r1)
( 99/103) Installing py3-wcwidth-pyc (0.6.0-r1)
(100/103) Installing py3-tabulate (0.10.0-r0)
(101/103) Installing py3-tabulate-pyc (0.10.0-r0)
(102/103) Installing py3-wheel (0.46.3-r1)
(103/103) Installing py3-wheel-pyc (0.46.3-r1)
Executing busybox-1.37.0-r31.trigger
OK: 339.7 MiB in 103 packages
>>> py3-cli_helpers: Fetching https://distfiles.alpinelinux.org/distfiles/py3-cli_helpers-2.12.0.tar.gz
/tmp/distfiles/py3-cli_helpers-2.12.0.tar.gz: OK
>>> py3-cli_helpers: Unpacking /tmp/distfiles/py3-cli_helpers-2.12.0.tar.gz...
2026-04-13 13:26:10,144 gpep517 INFO Building wheel via backend setuptools.build_meta:__legacy__
/usr/lib/python3.14/site-packages/setuptools/dist.py:765: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: OSI Approved :: BSD License

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
2026-04-13 13:26:10,152 root INFO running bdist_wheel
2026-04-13 13:26:10,157 root INFO running build
2026-04-13 13:26:10,157 root INFO running build_py
2026-04-13 13:26:10,158 root INFO creating build/lib/cli_helpers
2026-04-13 13:26:10,158 root INFO copying cli_helpers/config.py -> build/lib/cli_helpers
2026-04-13 13:26:10,158 root INFO copying cli_helpers/utils.py -> build/lib/cli_helpers
2026-04-13 13:26:10,158 root INFO copying cli_helpers/__init__.py -> build/lib/cli_helpers
2026-04-13 13:26:10,158 root INFO copying cli_helpers/compat.py -> build/lib/cli_helpers
2026-04-13 13:26:10,158 root INFO creating build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,158 root INFO copying cli_helpers/tabular_output/json_output_adapter.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO copying cli_helpers/tabular_output/output_formatter.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO copying cli_helpers/tabular_output/delimited_output_adapter.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO copying cli_helpers/tabular_output/tsv_output_adapter.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO copying cli_helpers/tabular_output/vertical_table_adapter.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO copying cli_helpers/tabular_output/__init__.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO copying cli_helpers/tabular_output/preprocessors.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO copying cli_helpers/tabular_output/tabulate_adapter.py -> build/lib/cli_helpers/tabular_output
2026-04-13 13:26:10,159 root INFO running egg_info
2026-04-13 13:26:10,160 root INFO creating cli_helpers.egg-info
2026-04-13 13:26:10,160 root INFO writing cli_helpers.egg-info/PKG-INFO
2026-04-13 13:26:10,160 root INFO writing dependency_links to cli_helpers.egg-info/dependency_links.txt
2026-04-13 13:26:10,160 root INFO writing requirements to cli_helpers.egg-info/requires.txt
2026-04-13 13:26:10,160 root INFO writing top-level names to cli_helpers.egg-info/top_level.txt
2026-04-13 13:26:10,160 root INFO writing manifest file 'cli_helpers.egg-info/SOURCES.txt'
2026-04-13 13:26:10,161 root INFO reading manifest file 'cli_helpers.egg-info/SOURCES.txt'
2026-04-13 13:26:10,161 root INFO reading manifest template 'MANIFEST.in'
2026-04-13 13:26:10,162 root WARNING warning: no previously-included files found matching '.pre-commit-config.yaml'
2026-04-13 13:26:10,162 root WARNING warning: no previously-included files found matching '.git-blame-ignore-revs'
2026-04-13 13:26:10,162 root INFO adding license file 'LICENSE'
2026-04-13 13:26:10,162 root INFO adding license file 'AUTHORS'
2026-04-13 13:26:10,162 root INFO writing manifest file 'cli_helpers.egg-info/SOURCES.txt'
2026-04-13 13:26:10,180 root INFO installing to build/bdist.linux-x86_64/wheel
2026-04-13 13:26:10,180 root INFO running install
2026-04-13 13:26:10,183 root INFO running install_lib
2026-04-13 13:26:10,184 root INFO creating build/bdist.linux-x86_64/wheel
2026-04-13 13:26:10,184 root INFO creating build/bdist.linux-x86_64/wheel/cli_helpers
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/config.py -> build/bdist.linux-x86_64/wheel/./cli_helpers
2026-04-13 13:26:10,184 root INFO creating build/bdist.linux-x86_64/wheel/cli_helpers/tabular_output
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/tabular_output/json_output_adapter.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/tabular_output/output_formatter.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/tabular_output/delimited_output_adapter.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/tabular_output/tsv_output_adapter.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/tabular_output/vertical_table_adapter.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/tabular_output/__init__.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,184 root INFO copying build/lib/cli_helpers/tabular_output/preprocessors.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,185 root INFO copying build/lib/cli_helpers/tabular_output/tabulate_adapter.py -> build/bdist.linux-x86_64/wheel/./cli_helpers/tabular_output
2026-04-13 13:26:10,185 root INFO copying build/lib/cli_helpers/utils.py -> build/bdist.linux-x86_64/wheel/./cli_helpers
2026-04-13 13:26:10,185 root INFO copying build/lib/cli_helpers/__init__.py -> build/bdist.linux-x86_64/wheel/./cli_helpers
2026-04-13 13:26:10,185 root INFO copying build/lib/cli_helpers/compat.py -> build/bdist.linux-x86_64/wheel/./cli_helpers
2026-04-13 13:26:10,185 root INFO running install_egg_info
2026-04-13 13:26:10,186 root INFO Copying cli_helpers.egg-info to build/bdist.linux-x86_64/wheel/./cli_helpers-2.12.0-py3.14.egg-info
2026-04-13 13:26:10,186 root INFO running install_scripts
2026-04-13 13:26:10,186 root INFO creating build/bdist.linux-x86_64/wheel/cli_helpers-2.12.0.dist-info/WHEEL
2026-04-13 13:26:10,186 wheel INFO creating '/tmp/src/cli_helpers-2.12.0/.dist/.tmp-un5i1o_b/cli_helpers-2.12.0-py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/__init__.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/compat.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/config.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/utils.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/__init__.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/delimited_output_adapter.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/json_output_adapter.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/output_formatter.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/preprocessors.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/tabulate_adapter.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/tsv_output_adapter.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers/tabular_output/vertical_table_adapter.py'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers-2.12.0.dist-info/licenses/AUTHORS'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers-2.12.0.dist-info/licenses/LICENSE'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers-2.12.0.dist-info/METADATA'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers-2.12.0.dist-info/WHEEL'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers-2.12.0.dist-info/top_level.txt'
2026-04-13 13:26:10,187 wheel INFO adding 'cli_helpers-2.12.0.dist-info/RECORD'
2026-04-13 13:26:10,187 root INFO removing build/bdist.linux-x86_64/wheel
2026-04-13 13:26:10,188 gpep517 INFO The backend produced .dist/cli_helpers-2.12.0-py3-none-any.whl
cli_helpers-2.12.0-py3-none-any.whl
============================= test session starts ==============================
platform linux -- Python 3.14.3, pytest-9.0.3, pluggy-1.6.0
rootdir: /tmp/src/cli_helpers-2.12.0
configfile: setup.cfg
testpaths: tests
collected 92 items

tests/tabular_output/test_delimited_output_adapter.py ...                [  3%]
tests/tabular_output/test_json_output_adapter.py .....                   [  8%]
tests/tabular_output/test_output_formatter.py .................          [ 27%]
tests/tabular_output/test_preprocessors.py .............sFFF........     [ 54%]
tests/tabular_output/test_tabulate_adapter.py ...                        [ 57%]
tests/tabular_output/test_tsv_output_adapter.py ...                      [ 60%]
tests/tabular_output/test_vertical_table_adapter.py ..                   [ 63%]
tests/test_cli_helpers.py .                                              [ 64%]
tests/test_config.py ..sss.............                                  [ 83%]
tests/test_utils.py ...............                                      [100%]

=================================== FAILURES ===================================
______________________________ test_style_output _______________________________

TwoFiftySixColor = None

    @pytest.mark.skipif(not HAS_PYGMENTS, reason="requires the Pygments library")
    def test_style_output(TwoFiftySixColor):
        """Test that *style_output()* styles output."""
    
        class CliStyle(Style):
            default_style = ""
            styles = {
                Token.Output.Header: "bold ansibrightred",
                Token.Output.OddRow: "bg:#eee #111",
                Token.Output.EvenRow: "#0f0",
            }
    
        headers = ["h1", "h2"]
        data = [["观音", "2"], ["Ποσειδῶν", "b"]]
    
        expected_headers = ["\x1b[91;01mh1\x1b[39;00m", "\x1b[91;01mh2\x1b[39;00m"]
        expected_data = [
            ["\x1b[38;5;233;48;5;7m观音\x1b[39;49m", "\x1b[38;5;233;48;5;7m2\x1b[39;49m"],
            ["\x1b[38;5;10mΠοσειδῶν\x1b[39m", "\x1b[38;5;10mb\x1b[39m"],
        ]
        results = style_output(data, headers, style=CliStyle)
    
>       assert (expected_data, expected_headers) == (list(results[0]), results[1])
E       AssertionError: assert ([['\x1b[38;5...\x1b[39;00m']) == ([['\x1b[38;5...\x1b[39;00m'])
E         
E         At index 0 diff: [['\x1b[38;5;233;48;5;7m观音\x1b[39;49m', '\x1b[38;5;233;48;5;7m2\x1b[39;49m'], ['\x1b[38;5;10mΠοσειδῶν\x1b[39m', '\x1b[38;5;10mb\x1b[39m']] != [['\x1b[38;5;233;48;5;255m观音\x1b[39;49m', '\x1b[38;5;233;48;5;255m2\x1b[39;49m'], ['\x1b[38;5;10mΠοσειδῶν\x1b[39m', '\x1b[38;5;10mb\x1b[39m']]
E         Use -v to get more diff

tests/tabular_output/test_preprocessors.py:216: AssertionError
_______________________ test_style_output_with_newlines ________________________

TwoFiftySixColor = None

    @pytest.mark.skipif(not HAS_PYGMENTS, reason="requires the Pygments library")
    def test_style_output_with_newlines(TwoFiftySixColor):
        """Test that *style_output()* styles output with newlines in it."""
    
        class CliStyle(Style):
            default_style = ""
            styles = {
                Token.Output.Header: "bold ansibrightred",
                Token.Output.OddRow: "bg:#eee #111",
                Token.Output.EvenRow: "#0f0",
            }
    
        headers = ["h1", "h2"]
        data = [["观音\nLine2", "Ποσειδῶν"]]
    
        expected_headers = ["\x1b[91;01mh1\x1b[39;00m", "\x1b[91;01mh2\x1b[39;00m"]
        expected_data = [
            [
                "\x1b[38;5;233;48;5;7m观音\x1b[39;49m\n\x1b[38;5;233;48;5;7m"
                "Line2\x1b[39;49m",
                "\x1b[38;5;233;48;5;7mΠοσειδῶν\x1b[39;49m",
            ]
        ]
        results = style_output(data, headers, style=CliStyle)
    
>       assert (expected_data, expected_headers) == (list(results[0]), results[1])
E       AssertionError: assert ([['\x1b[38;5...\x1b[39;00m']) == ([['\x1b[38;5...\x1b[39;00m'])
E         
E         At index 0 diff: [['\x1b[38;5;233;48;5;7m观音\x1b[39;49m\n\x1b[38;5;233;48;5;7mLine2\x1b[39;49m', '\x1b[38;5;233;48;5;7mΠοσειδῶν\x1b[39;49m']] != [['\x1b[38;5;233;48;5;255m观音\x1b[39;49m\n\x1b[38;5;233;48;5;255mLine2\x1b[39;49m', '\x1b[38;5;233;48;5;255mΠοσειδῶν\x1b[39;49m']]
E         Use -v to get more diff

tests/tabular_output/test_preprocessors.py:244: AssertionError
_______________________ test_style_output_custom_tokens ________________________

TwoFiftySixColor = None

    @pytest.mark.skipif(not HAS_PYGMENTS, reason="requires the Pygments library")
    def test_style_output_custom_tokens(TwoFiftySixColor):
        """Test that *style_output()* styles output with custom token names."""
    
        class CliStyle(Style):
            default_style = ""
            styles = {
                Token.Results.Headers: "bold ansibrightred",
                Token.Results.OddRows: "bg:#eee #111",
                Token.Results.EvenRows: "#0f0",
            }
    
        headers = ["h1", "h2"]
        data = [["1", "2"], ["a", "b"]]
    
        expected_headers = ["\x1b[91;01mh1\x1b[39;00m", "\x1b[91;01mh2\x1b[39;00m"]
        expected_data = [
            ["\x1b[38;5;233;48;5;7m1\x1b[39;49m", "\x1b[38;5;233;48;5;7m2\x1b[39;49m"],
            ["\x1b[38;5;10ma\x1b[39m", "\x1b[38;5;10mb\x1b[39m"],
        ]
    
        output = style_output(
            data,
            headers,
            style=CliStyle,
            header_token=Token.Results.Headers,
            odd_row_token=Token.Results.OddRows,
            even_row_token=Token.Results.EvenRows,
        )
    
>       assert (expected_data, expected_headers) == (list(output[0]), output[1])
E       AssertionError: assert ([['\x1b[38;5...\x1b[39;00m']) == ([['\x1b[38;5...\x1b[39;00m'])
E         
E         At index 0 diff: [['\x1b[38;5;233;48;5;7m1\x1b[39;49m', '\x1b[38;5;233;48;5;7m2\x1b[39;49m'], ['\x1b[38;5;10ma\x1b[39m', '\x1b[38;5;10mb\x1b[39m']] != [['\x1b[38;5;233;48;5;255m1\x1b[39;49m', '\x1b[38;5;233;48;5;255m2\x1b[39;49m'], ['\x1b[38;5;10ma\x1b[39m', '\x1b[38;5;10mb\x1b[39m']]
E         Use -v to get more diff

tests/tabular_output/test_preprocessors.py:277: AssertionError
=========================== short test summary info ============================
FAILED tests/tabular_output/test_preprocessors.py::test_style_output - Assert...
FAILED tests/tabular_output/test_preprocessors.py::test_style_output_with_newlines
FAILED tests/tabular_output/test_preprocessors.py::test_style_output_custom_tokens
=================== 3 failed, 85 passed, 4 skipped in 0.17s ====================
>>> ERROR: py3-cli_helpers: check failed
>>> ERROR: py3-cli_helpers: rootbld failed
```

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,21 +1,28 @@
 # Changelog
 
+## Version 2.13.0
+
+(released on 2026-04-13)
+
+- Pin Pygments library to older version v2.19.2 to fix tests.
+- Fix dates in changelog.
+
 ## Version 2.12.0
 
-(released on 2025-03-21)
+(released on 2026-03-21)
 
 - Probe tabulate version for `preserve_whitespace` keyword support.
 
 ## Version 2.11.0
 
-(released on 2025-03-05)
+(released on 2026-03-05)
 
 - Support and require version 0.10.x of `tabulate`.
 - Pin `tabulate` version more tightly.
 
 ## Version 2.10.1
 
-(released on 2025-02-18)
+(released on 2026-02-18)
 
 - Update license dates.
 - Change `master` branch references to `main`.

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -2,7 +2,7 @@ autopep8==1.3.3
 codecov==2.1.13
 coverage==4.3.4
 black>=20.8b1
-Pygments>=2.4.0
+Pygments~=2.19.2
 pytest==7.4.3
 pytest-cov==2.4.0
 Sphinx==1.5.5

--- a/setup.py
+++ b/setup.py
@@ -40,7 +40,7 @@ setup(
         "tabulate[widechars] ~= 0.10.0",
     ],
     extras_require={
-        "styles": ["Pygments >= 1.6"],
+        "styles": ["Pygments ~= 2.19.2"],
     },
     python_requires=">=3.6",
     classifiers=[

